src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  result = days * 40;
+
+  if (days >= 3) {
+    result = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    result = days * 40 - 50;
+  }
```

Comment:
a lot of useless actions
```suggestion
  let result = days * 40;

  if (days >= 3) {
    result -= 20;
  }

  if (days >= 7) {
    result -= 50;
  }
```